src/actions/data.js

```diff
@@ -2,19 +2,19 @@
 import type { Dispatch } from "redux";
 import { getCmsData, updateCmsData } from "../integrations/cms";
 import type { SavedData } from "../integrations/cms";
-import type { StandardAction } from "./";
 
-type CmsActionType =
-  | "REQUEST_CMS_DATA"
-  | "RECEIVE_CMS_DATA"
-  | "REQUEST_UPDATE_CMS_DATA";
-
-export type CmsAction = StandardAction<CmsActionType, SavedData>;
```

Comment:
Instead of using the StandardAction, which makes the payload optional, I have explicitly stated the sape of each kind of action. This allows us to access deeper into the data property with checking if it exists.